sdk/communication/communication-sms/README.md

```diff
@@ -113,9 +120,16 @@ for (const sendResult of sendResults) {
 To check if the recipients are in the Opt Out list, call the `check` function from the `SmsClient.optOuts` with a list of recipient phone numbers.
 
 ```ts snippet:ReadmeSampleOptOutCheck
+import { DefaultAzureCredential } from "@azure/identity";
+import { SmsClient } from "@azure/communication-sms";
+
+const endpoint = "https://<resource-name>.communication.azure.com";
+const credential = new DefaultAzureCredential();
+const client = new SmsClient(endpoint, credential);
+
 const optOutCheckResults = await client.optOuts.check(
   "<from-phone-number>", // Your E.164 formatted phone number used to send SMS
-  ["<to-phone-number-1>", "<to-phone-number-2>"], // E.164 formatted recipient phone numbers
+  ["<to-phone-number-1>", "<to-phone-number-2>"],
```

Comment:
It might be better to update the snippet to include this comment too.
```suggestion
  ["<to-phone-number-1>", "<to-phone-number-2>"], // E.164 formatted recipient phone numbers
```